starter_code/index.html

```diff
@@ -1,18 +1,50 @@
-<!DOCTYPE html>
 <html lang="en">
 	<head>
+			<link href='//fonts.googleapis.com/css?family=Open+Sans:400italic,600italic,700italic,400,600,700' rel='stylesheet' type='text/css'>
     	<meta charset="utf-8">
     	<meta name="description" content="Your description goes here">
     	<meta name="keywords" content="one, two, three">
 
-		<title>Blank Template</title>
+		<title>Relaxr</title>
 
 		<!-- external CSS link -->
-		<link rel="stylesheet" href="css/reset.css">
 		<link rel="stylesheet" href="css/style.css">
 
 	</head>
 	<body>
-		<h1>Blank Template</h1>
+		<header>
+			<h1 >Relaxr</h1>
+			<h2>Get piece of mind with a single tap</h2>
+			<button  >Get it Now</button>
+		</header>
+		<main>
+		<div class="benefits">
+				<h3  >Benefits</h3>
+				<p id="theperfec">The perfect personal assistant. Relaxr 
+					does your job for you so you can enjoy 
+					life as it is meant to be.</p>
+				<ul>
+					<li>Schedule meetings for you</li>
+					<li>Excel automation to complete your work for you</li>
+					<li>Responds to emails on your behalf</li>
+					<li>Does all your work for you with our revolutionary AI technology</li>
+				</ul>
+			</div>
+			
+				<p  class="experience">“Relaxr changed my life. I’ve been able to travel the world, spend limited time working and my boss keeps thanking me for crushing work.”
+				<strong>- Amanda, Intuit</strong> </p>
```

Comment:
Try to separate the sections in your page, the above two elements would be easier to deal with if they were in a div and you applied your style to it